cucumber/api/features/login.feature

```diff
@@ -5,12 +5,87 @@ Feature: Exchange a role's password for its API key
 
   Background:
     Given I create a new user "alice"
+    Given I have host "app"
 
   Scenario: Password can be used to obtain API key
     Given I set the password for "alice" to "My-Password1"
     When I can GET "/authn/cucumber/login" with username "alice" and password "My-Password1"
     Then the HTTP response content type is "text/plain"
     And the result is the API key for user "alice"
+    And there is an audit record matching:
+    """
+      <84>1 * * conjur * authn
+      [auth@43868 authenticator="authn" service="default:webservice:conjur/authn]
+      [subject@43868 role="cucumber:user:alice" user="alice"]
+      [action@43868 operation="login" result="success"]
+      cucumber:user:alice successfully logged in with authenticator authn service default:webservice:conjur/authn
+    """
+  Scenario: Password can be used by host to obtain API key
+    Given I set the password for "host/app" to "My-Password1"
+    When I can GET "/authn/cucumber/login" with username "host/app" and password "My-Password1"
+    Then the HTTP response content type is "text/plain"
+    And the result is the API key for host "app"
+    And there is an audit record matching:
+    """
+      <84>1 * * conjur * authn
+      [auth@43868 authenticator="authn" service="default:webservice:conjur/authn]
+      [subject@43868 role="cucumber:host:app" user="host/app"]
+      [action@43868 operation="login" result="success"]
+      cucumber:host:app successfully logged in with authenticator authn service default:webservice:conjur/authn
+    """
+
+  Scenario: Wrong password cannot be used to obtain API key
+    When I GET "/authn/cucumber/login" with username "alice" and password "Wrong-Password"
+    Then the HTTP response status code is 401
+    And there is an audit record matching:
+    """
+      <84>1 * * conjur * authn
+      [subject@43868 role="CyberArk:user:alice" user="not-found"]
```

Comment:
what am i missing here? is the user `alice` not found or in the password incorrect? hw come the role is `CyberArk:user:alice` and not `cucumber:user:alice`?

Reply:
copy-paste mistake from other audits. will fix